ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+puts "How many employes will be processed?" 
+employes = gets.chomp
+
+puts "What is your name?"
+vampire_name = gets.chomp
+
+puts "How old are you? What year were you born in?"
+age = gets.chomp
+
+puts "Our company cafeteria serves garlic bread. Should we order you some?"
+garlic = gets.chomp
+
+puts "Would you like to enroll the company’s health insurance?"
+insurance = gets.chomp
+
+
+garlic= true
+vampire_name = true
+insurance = true
+age = true
+
+if #{age} && #{garlic} || #{insurance} == true 
```

Comment:
It looks like you are trying to interpolate a sting here.  Instead just go ahead and set the variable to be evaluated.

```
if age && garlic || insurance == true
```